requirements/default.in

```diff
@@ -57,14 +57,14 @@ parameterized~=0.7.4
 premailer~=3.7.0
 puente~=0.5.0
 py-wikimarkup~=2.0.1
-pyOpenSSL~=19.1.0
+pyOpenSSL~=20.0.0
 pyquery~=1.2.9
 python-dateutil~=2.8.1
 python-decouple~=3.3
 python-memcached~=1.59
 pytz~=2020.1
 redis~=3.5.2
-requests[security]~=2.23.0
```

Comment:
Was this what fixed the pip deps error?

Reply:
If I understand this correctly (big if), the new resolver that was introduced in pip v20.3+, changed the way constraints work. Constraints are loaded prior to the actual resolver so when pip sees that `extra_require`, it tries to deal with it as a dependency but the hash checking mode is complaining that those extras are not pinned.